stories/Welcome.mdx

```diff
@@ -1,7 +1,158 @@
-import './styles.css';
+import './welcome-styles.css';
+import gridironLogo from './assets/gridironSurvivorLogo.svg';
+import appwriteLogo from './assets/appwriteLogo.svg';
+import blogRecorderLogo from './assets/blogRecorderLogo.svg';
+import frontendMentorLogo from './assets/frontendMentorLogo.svg';
+import gitkrakenLogo from './assets/gitkrakenLogo.svg';
+import pastelLogo from './assets/pastelLogo.svg';
+import Image from 'next/image';
+import { ArrowRight } from 'lucide-react';
+import { Github } from 'lucide-react';
 
-<h1>Welcome</h1>
+import { Button } from '../components/Button/Button.tsx';
 
-<div className="doc_nav_links">
-  <p>[Application Setup](?path=/docs/about-application-setup--docs)</p>
-</div>
+<html>
+  <body class="dark bg-background text-muted-foreground max-w-screen-lg mx-auto text-pretty flex flex-col space-y-36 pb-36">
+    <header class="header flex justify-center align-middle">
+      <a href="https://www.gridironsurvivor.com" target="_blank" rel="noreferrer" class="w-1/4 h-fit">
+        <Image class="w-full" src={gridironLogo} alt='Gridiron Survivor Football Logo' />
+      </a>
+    </header>
+    <section class="hero space-y-12">
+      <h1>What is Gridiron Survivor?</h1>
+      <p class="text-2xl text-pretty max-w-screen-lg mx-auto">
+        Gridiron Survivor is the ultimate survivor league app. Track your picks, see how you stack up against your competition, and hope to be crowned champion! Developed by a talented team of Junior Developers led by Senior Software Engineer Shashi Lo, Gridiron Survivor is the perfect app for any football fan looking to take their survivor league to the next level.
```

Comment:
`Track your picks, see how you stack up against your competition, and hope to be crowned champion!` Please remove the comma here after competition, this is currently incorrect.